src/components/shared/TreeView.js

```diff
@@ -56,37 +56,100 @@ export type Column<DisplayData: Object> = {|
 type TreeViewHeaderProps<DisplayData: Object> = {|
   +fixedColumns: Column<DisplayData>[],
   +mainColumn: Column<DisplayData>,
+  +onSort: (number) => void,
+  +columnSortDirections?: Array<boolean | null>,
 |};
 
-const TreeViewHeader = <DisplayData: Object>({
-  fixedColumns,
-  mainColumn,
-}: TreeViewHeaderProps<DisplayData>) => {
-  if (fixedColumns.length === 0 && !mainColumn.titleL10nId) {
-    // If there is nothing to display in the header, do not render it.
-    return null;
+// columns start at 1
+export class ColumnSortState {
```

Comment:
> So instead of keeping an up-down arrow for every column, we should only keep one arrow in a table. That one arrow should indicate which column it's sorted by and if it's ascending or descending.

It allows to cascade sorting. But I see, that it's probably better to keep it out for now.